pom.xml

```diff
@@ -160,6 +149,25 @@ Use is subject to <a href="{@docRoot}/doc-files/speclicense.html" target="_top">
                   </execution>
               </executions>
           </plugin>
+      <plugin>
+        <groupId>org.moditect</groupId>
+        <artifactId>moditect-maven-plugin</artifactId>
+        <version>1.0.0.RC1</version>
+        <executions>
+          <execution>
+            <id>add-module-infos</id>
+            <phase>package</phase>
+            <goals>
+              <goal>add-module-info</goal>
+            </goals>
+            <configuration>
+              <module>
+                <moduleInfoFile>src/main/java9/module-info.java</moduleInfoFile>
```

Comment:
I think other specs put this files under src/main/java (see [here](https://github.com/eclipse-ee4j/yasson/tree/master/src/main/java))

Reply:
Can we follow the same convention?

Reply:
yasson is already on Java 9. Java 8's javac would choke on this file, if it tried to compile it. Therefore, the multi-version layout is required.

Reply:
Yea putting this into `java9` (or `module` or something) is the right thing to do here, as we want to build the main source with Java 8.